cmd/example-gcp/main.go

```diff
@@ -72,6 +73,21 @@ func main() {
 		_, _ = w.Write([]byte(fmt.Sprintf("%d", idx)))
 	})
 
+	// TODO: remove this proxy
+	serveGCS := func(w http.ResponseWriter, r *http.Request) {
+		resource := strings.TrimLeft(r.URL.Path, "/")
+		b, err := storage.Get(r.Context(), resource)
+		if err != nil {
+			klog.V(1).Infof("Get: %v", err)
+			w.WriteHeader(http.StatusBadRequest)
```

Comment:
Classic denial of any responsibility here :-)

Reply:
It's probably your fault, to be fair. :)